tweb/src/main/resources/templates/index.html

```diff
@@ -83,7 +84,37 @@ <h1 class="home-title">
                 </div>
             </div>
         </div>
+        <div class="container my-5 px-5">
```

Comment:
One of the requirements of this issue is:
> A separator to show the beginning of the new section (1px grey line)

this still needs to be added

Reply:
Consider it done, and please share the transparant image with me :)

Reply:
done